src/TMLeague/wwwroot/data/leagues/ptl/seasons/s2/divisions/d1.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Division 1",
+  "judge": null,
+  "players": [
+    "Joao Gomes",
+    "spartan109",
+    "sicre",
+    "jpacmelo",
+    "Julian Snow",
+    "diogobensabat",
+    "Jimmymarshal",
+    "Ser Santos",
+    "TurtleHermit",
+    "ZecasNando"
+  ],
+  "games": [
+	256246,
+	256253,
+	256247,
+	256248,
+	256249,
+	256254,
+	256250,
+	256255,
+	256251,
+	256258
+  ],
+  "penalties": [
+	{
+	  "player": "ZecasNando",
+	  "points": 6,
+	  "details": "-4 Absence ; -2 MPM"
+	},
+	{
+	  "player": "spartan109",
+	  "points": 24,
```

Comment:
Why 0 points is a minimum? ;)

Reply:
We've decided as a group not to be so harsh and define that noone should get negative points for the whole season that would impact other seasons that significantly.